### ISSUE TITLE
Bug: NaN Calc world cframe

### ISSUE THREAD

**Issue by Nezhull0** [smartbone-org/SmartBone-2#38]
▶ If you see this report this as a bug, (NaN Calc world cframe) (x4960)  -  Client - Utilities:262

This bug happens pretty consistently, I am not exactly sure why.
 I am using Version 0.4.0

**Comment by JakeyWasTaken** [comment]
Could you tell me if this happens on 0.3.1 aswell with the same setup you're currently using.

**Comment by Nezhull0** [comment]
No, I haven't noticed the bug reappear. My apologies, I should have read your description more thoroughly.

**Comment by bzdogs** [comment]
From what i've seen it happens when you set the CFrame of the object via a WeldConstraint.

Thats what causes this for my use case.

**Comment by bzdogs** [comment]
Issue was resolved when reverting to 0.3.1

**Comment by JakeyWasTaken** [comment]
Could you get back to me on if this still occurs in 0.5.0?

**Comment by JakeyWasTaken** [comment]
Gonna close this for now
